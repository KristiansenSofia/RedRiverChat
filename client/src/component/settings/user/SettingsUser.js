import React, { Component } from 'react';

import {settingsUserStyles} from "../../../styles/SettingsStyles";
import '../../../styles/Styles.css'

class SettingsUser extends Component {
    render() {
        return (
            <div className="SettingsUser">
                <p>User settings</p>
            </div>
        );
    }
}

export default SettingsUser;