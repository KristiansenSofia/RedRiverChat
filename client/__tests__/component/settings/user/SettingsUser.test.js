import React from 'react';
import SettingsUser from '../../../../src/component/settings/user/SettingsUser';
import renderer from 'react-test-renderer';

test('SettingsUser renders without crashing', () => {
  const tree = renderer.create(
    <SettingsUser state = {{ userInfo : { username: 'username' } }}/>
  );
  expect(tree.toJSON()).toMatchSnapshot();
});
