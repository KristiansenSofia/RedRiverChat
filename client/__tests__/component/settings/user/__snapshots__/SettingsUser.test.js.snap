// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsUser renders without crashing 1`] = `
<div
  className="SettingsUser"
>
  <p>
    User settings
  </p>
</div>
`;
